esti/ops/docker-compose-dynamodb.yaml

```diff
@@ -0,0 +1,88 @@
+version: '3'
```

Comment:
We are running esti tests on local dynamodb instance? Not sure it will hold when all packages are migrated...

Reply:
In terms of what? Throughput? Consistency guarantees? Shouldn't we wait to figure it out?

Reply:
I'm also reluctant to pay for the real thing for our CI if it's not absolutely a must